### ISSUE TITLE
False positive for ComposeContentEmitterReturningValues

### ISSUE THREAD

**Issue by ansman** [slackhq/compose-lints#166]
It seems that, in at least some cases, `ComposeContentEmitterReturningValues` is incorrectly reported when `ComposeMultipleContentEmitters` is also detected.

Here is an example:
```kotlin
@Composable
fun Example(modifier: Modifier = Modifier) {
    Box(modifier = modifier)
    Box(Modifier)
}
```
<img width="996" alt="image" src="https://github.com/slackhq/compose-lints/assets/102883/7860f411-978e-4bb9-8479-ed6001c87080">

Removing the second `Box` removes both errors.

Also, adding `@SuppressLint("ComposeContentEmitterReturningValues", "ComposeMultipleContentEmitters")` does not suppress the issue.

Versions:
- compose-lints 1.2.0
- AGP 8.2.0-alpha16
- Android Studio Hedgehog | 2023.1.1 Canary 16
- Kotlin 1.8.21

**Comment by StylianosGakis** [comment]
The expected result here would be to only report the "do-not-emit-multiple-pieces-of-content" issue right? Since that one is in fact happening here.
Because I am experiencing the same thing now, and wanted to double check.

**Comment by ansman** [comment]
Yes, the composable is returning Unit/void so it's not returning a value

**Comment by ZacSweers** [comment]
I'm unable to reproduce this on `main`. Here's what I tested https://github.com/slackhq/compose-lints/tree/z/repro166

**Comment by ansman** [comment]
Feel free to close then, It happened to 100% of the time for me so if you can't repro it it's likely fixed
